src/componentes/ButtonMenu/ButtonMenu.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './ButtonMenu.css';
+
+function ButtonMenu(props) {
+  return (
+    <div 
+    className={`button-menu ${props.active ? 'active' : ''}`.trimEnd() }>
```

Comment:
esta clase se puede mover a una variable separada

```
const isButtonActive = props.active ? 'active' : '';
```
 y en esta linea se puede simplemente usar esa variable
```suggestion
    className={`button-menu ${isButtonActive}`.trimEnd() }>
```
Es buena práctica que este tipo de operaciones sean ejecutadas fuera del HTML. Hace el template mas legible.